cypress.config.js

```diff
@@ -0,0 +1,31 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  screenshotsFolder: 'cypress/snapshots/actual',
+  trashAssetsBeforeRuns: true,
+  video: false,
+  env: {
+    failSilently: false,
+  },
+  e2e: {
+    // We've imported your old cypress plugins here.
```

Comment:
This comments automatically generated after migrating to cypress version 10. So maybe I should remove this lines.